### ISSUE TITLE
Allow wrapping on words in Floating Window

### ISSUE THREAD

**Issue by parmardiwakar150** [luckasRanarison/nvim-devdocs#7]
This is my config for devdocs

```lua
require("nvim-devdocs").setup({
	dir_path = vim.fn.stdpath("data") .. "/devdocs",
	telescope = {},
	telescope_alt = {
		layout_config = {
			width = 75,
		},
	},
	float_win = {
		relative = "editor",
		height = math.floor(vim.o.lines * 0.7),
		width = math.floor(vim.o.columns * 0.8),
		border = "rounded",
	},
	wrap = true,
	ensure_installed = {},
})

```
I am opening DevDocs in a floating window, but some lines contain more characters than it can fit on the screen, causing the text to be hidden. By enabling the `wrap = true `setting, I am able to view the complete text, but it is wrapping based on the characters.

<img width="1428" alt="image" src="https://github.com/luckasRanarison/nvim-devdocs/assets/31608288/b30be93d-ee74-4a71-a9c4-e5aada78d92a">

I would like the text to wrap based on words. If I run `set linebreak` for the floating window, then it fixes the issues.
<img width="1448" alt="image" src="https://github.com/luckasRanarison/nvim-devdocs/assets/31608288/9a254637-2ed0-40ab-b750-18e490d4c118">

Is there any way to set this by default for the floating window? If there isn't any, can you please add some config parameter for it?

**Comment by luckasRanarison** [comment]
Thanks for the report, I didn't know that wrap worked thay way, it'd make sense to add linebreak by default if wrap is set to true. I'll change this right away.

**Comment by luckasRanarison** [comment]
I fixed it in aa884e3

**Comment by parmardiwakar150** [comment]
Thanks a lot
